pages/1.13/security/oss/authentication/out-of-band-verification/index.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: layout.pug
+navigationTitle:  Out-of-band Token Verification
+title: Out-of-band Token Verification
+excerpt: Verifying DC/OS Authentication tokens out-of-band
+menuWeight: 20
+
+---
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+Other services can authenticate incoming requests on behalf of the DC/OS [Identity and Access Manager (IAM)](/1.13/overview/architecture/components/#dcos-iam) component, using public key cryptography. This works if the authentication token presented by the client has been signed by the IAM with its private key.
+
+## Bouncer JSON Web Key Set (JWKS) endpoint
+The Bouncer's JWKS endpoint (`/auth/jwks`) provides the public key details required for verifying the signature of type RS256 JWTs issued by Bouncer. The JSON document data structure emitted by that endpoint is compliant with [RFC 7517](https://tools.ietf.org/html/rfc7517). Within that data structure, the public key is parameterized according to [RFC 7518](https://tools.ietf.org/html/rfc7518).
+
+Here is an example response:
+
+```json
+curl -k https://<host-ip/acs/api/v1/auth/jwks
+{
+  "keys": [
+    {
+      "e": "AQAB",
+      "use": "sig",
+      "n": "7dYvibxUngEyfdut1uSYbRCCP5dT5MQyMfLyy_6o5x8PD-fUMgkm0vGUJAUoKimnkZ85aUmswaU3yAxQiZ8yeaoSpgUR4WJCRhOIEJ6Oyq4mjK06vr9-wJj5gVXDBaqbxD0yhgzMHEDyxg3EFOJ2ve73Vkg4p7pygA4fI_de1Bs6n68Hwt9LJ7B-fPg0PU8IdPe_4dYNuHT09KGxWSlq3m4KSvNxPIGQ8nNK9H3gjQaoBT9-hDXfsAgrQo7GenXRZTYW13KATtRAR5Vtd177iEeVefbK3HRj9IfYjYPnlBP2CZv_YIK-9H_33JPXxlDTFgI92l_JKRF-fPSa1EEkIw",
+      "alg": "RS256",
+      "kid": "49f795b26f80bec01f44b0f52e6ba6459ee2048fbb342f861f1a4e8ed4ebcb7f",
+      "kty": "RSA"
+    }
+  ]
+}
+```
+
+## Constructing the public key from the JWKS data
+The two parameters that fully define an RSA public key are the modulus (`n`) and the exponent (`e`). Both are integers encoded using Base64 as specified in [RFC 7518](https://tools.ietf.org/html/rfc7518#section-6.3)
+
+Use the tool of your choice to generate the public key representation that you will need to validate the authentication token. Here is a Python example based on the cryptography module (which uses OpenSSL as its back-end). This example generates a public key object from a given exponent and modulus directly.
```

Comment:
Rewrite line #37 as:
`Use the tool of your choice to generate the public key representation that is required to validate the authentication token. Here is a Python example based on the cryptography module (which uses OpenSSL as its back-end) is used to generates a public key object from a given exponent and modulus directly.`